.github/workflows/pack_publish.yml

```diff
@@ -407,9 +389,285 @@ jobs:
+          Write-Error "No installer found in artifact"
+          exit 1
+        }
+        
+        echo "Found installer: $($installer.FullName)"
+        $fileInfo = Get-Item $installer.FullName
+        $fileSizeMB = [math]::Round($fileInfo.Length / 1MB, 2)
+        echo "File size: $($fileInfo.Length) bytes ($fileSizeMB MB)"
+      shell: pwsh
+    
+    - name: Install Manticore
+      timeout-minutes: 5
+      run: |
+        echo "=== INSTALLING MANTICORE ==="
+        
+        # Extract tar if needed and not already extracted
+        $tarFile = Get-ChildItem ./installer/*.tar -ErrorAction SilentlyContinue | Select-Object -First 1
+        if ($tarFile) {
+          $buildDir = Get-ChildItem ./installer/build -ErrorAction SilentlyContinue
+          if (-not $buildDir) {
+            tar -xf $tarFile.FullName -C ./installer
+          }
+        }
+        
+        # Look for installer recursively
+        $installer = Get-ChildItem ./installer -Filter "*.exe" -Recurse -ErrorAction SilentlyContinue | Select-Object -First 1
+        
+        if (-not $installer) {
+          echo "ERROR: Installer not found!"
+          Get-ChildItem ./installer -Recurse
+          exit 1
+        }
+        
+        echo "Starting silent installation with /S flag..."
+        echo "Installing from: $($installer.FullName)"
+        $process = Start-Process -FilePath $installer.FullName -ArgumentList "/S" -Wait -PassThru -NoNewWindow
+        
+        echo "Installation process exited with code: $($process.ExitCode)"
+        
+        if ($process.ExitCode -eq 0) {
+          echo "Installation completed successfully (exit code 0)"
+        } elseif ($process.ExitCode -eq 1) {
+          echo "Installation returned exit code 1 (possible download error)"
+          echo "This may happen if installer couldn't download additional components"
+        } elseif ($process.ExitCode -eq 6) {
+          echo "Installation returned exit code 6 (Docker pull failed for executor)"
+          echo "Core components should still be installed"
+        } else {
+          echo "Installation exited with unexpected code $($process.ExitCode)"
+        }
+        
+        # Wait for installation to complete
+        Start-Sleep -Seconds 30
+      shell: pwsh
+    
+    - name: Verify Installation
+      run: |
+        echo "=== VERIFYING INSTALLATION ==="
+        
+        # Check installation paths
+        $possiblePaths = @(
+          "C:\Program Files (x86)\Manticore",
+          "C:\Program Files\Manticore",
+          "C:\Manticore"
+        )
+        
+        $foundPath = $null
+        foreach ($path in $possiblePaths) {
+          if (Test-Path $path) {
+            echo "Found Manticore installation at: $path"
+            $foundPath = $path
+            
+            # Check for key executables
+            $searchd = Get-ChildItem -Path $path -Filter "searchd.exe" -Recurse -ErrorAction SilentlyContinue | Select-Object -First 1
```

Comment:
We don't need searchd.exe anywhere inside the path. It should have an exact path, so remove the `-Recurse`. Same for `$indexer`